CMakeLists.txt

```diff
@@ -75,7 +76,7 @@ include_directories(
     ${CMAKE_SOURCE_DIR}/proof
     ${CMAKE_SOURCE_DIR}/models
     ${CMAKE_SOURCE_DIR}/itehandler
-    ${GMP_INCLUDE_DIR}
+    ${GMP_INCLUDE_DIRS}
```

Comment:
Why is this in plural?  It causes problems on my mac in plural but works in singular.

Reply:
I read somewhere, and [this](https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html) CMake documentation confirms it, that the plural version should be the output of find_package, while the singular version is the input (a hint where to look). But maybe FindGMP module does not follow this convention.

Reply:
I have pushed a fix, can you try it?

Reply:
Works now both in cluster and my mac.